compiler/rustc_codegen_ssa/src/traits/builder.rs

```diff
@@ -405,6 +405,31 @@ pub trait BuilderMethods<'a, 'tcx>:
         index: Self::Value,
     );
 
+    fn mcdc_parameters(
+        &mut self,
+        fn_name: Self::Value,
+        hash: Self::Value,
+        bitmap_bytes: Self::Value,
+    ) -> Self::Value;
+
+    fn mcdc_condbitmap_update(
+        &mut self,
+        fn_name: Self::Value,
+        hash: Self::Value,
+        cond_loc: Self::Value,
+        mcdc_temp: Self::Value,
+        bool_value: Self::Value,
+    );
+
+    fn mcdc_tvbitmap_update(
+        &mut self,
+        fn_name: Self::Value,
+        hash: Self::Value,
+        bitmap_bytes: Self::Value,
+        bitmap_index: Self::Value,
+        mcdc_temp: Self::Value,
+    );
+
```

Comment:
These are never invoked from codegen_ssa, so they don't need to exist here. Please move them to codegen_llvm

Reply:
Done, these functions are now removed from `BuilderMethod` as well as codegen_gcc.